ReSharper.FSharp/src/FSharp.TypeProviders.Host.NetCore/FSharp.TypeProviders.Host.NetCore.csproj

```diff
@@ -0,0 +1,47 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>netcoreapp3.1</TargetFramework>
+    </PropertyGroup>
+
+    <PropertyGroup>
+        <AssemblyName>JetBrains.ReSharper.Plugins.FSharp.TypeProviders.Host.NetCore</AssemblyName>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <CSFile Include="../FSharp.TypeProviders.Host/src/**/*.cs"/>
```

Comment:
What this item group is needed for?

Reply:
FSharp.TypeProviders.Host.NetCore must use the same sources as FSharp.TypeProviders.Host, so referencing them like this

Reply:
Thanks, I see.